src/types/policies/EmissionPrice.jl

```diff
@@ -12,6 +13,15 @@ Emission Price - A price on a certain emission for a given set of generators.
 * `ref_year_col`: Column name to use as reference year for min and max above. Must be a year column. If this is :year_on, then the years_after_ref filters will filter gen age. If this is :year_retrofit, the the years_after_ref filters will filter by time since retrofit. This is rarely used in real policy, so be careful if changing from default value
 * `gen_filters`: OrderedDict of generator filters
 * `hour_filters`: OrderedDict of hour filters
+
+### Table Column Added:
+* `(:gen, :pol.name)` - emissions price per MWh generated for each policy
+* `(:gen, :<name>_capex_adj)` - Adjustment factor added to the obj function as a PerMWCapInv term to account for emisprc payments that do not continue through the entire econ lifetime of a generator
+
+### Results Formulas:
+* `(:gen, :cost_name)` - the cost of the policy 
```

Comment:
I think it's a variable called `cost_name`, could you find how that variable is made?  I.e. `<name>_cost` or similar.